9_ninth_hw/1_first_task.py

```diff
@@ -0,0 +1,138 @@
+"""
+Create a class describing any company. For example, Toshiba or Apple
+
+I will evaluate the completeness of the described class and will withdraw points for everything unnecessary.
+I want to see clean code. I expect to see clean code with annotations.
+So far, no first-class connections with the second. In all methods, I expect to see docstrings with a sane description.
+"""
+from datetime import date, timedelta, datetime
+
+
+class Apple:
+    BRAND = "Apple"
+
+    def __init__(self, __website: str, __address: str, __employees: int):
+        self.__website = __website
+        self.__address = __address
+        self.__employees = __employees
+
+    @property
+    def employees(self):
+        return self.__employees
+
+    @employees.setter
+    def employees(self, new_employees_amount: int):
+        self.__employees = new_employees_amount
+
+    def new_employee(self, new_employees_num: int):
+        """
+        :param new_employees_num: update number of employees by the new employees
+        :return: new number of employees
+        """
+        self.__employees += new_employees_num
```

Comment:
So if I pass -10000000000000000 the value goes to some interesting value. 
Please add different checks